.gitignore

```diff
@@ -25,3 +25,6 @@ out/
 
 # unencrypted service account creds
 .cache-credentials.json
+
+# No Kotlin lib/
+lib/
```

Comment:
this shouldn't be necessary if the IntelliJ project is imported from bazel

Reply:
ok going to close this and try it again after all the merged changes